src/app/window-handler.ts

```diff
@@ -918,6 +918,26 @@ export class WindowHandler {
         delete this.windows[key];
     }
 
+    /**
+     * Executes the given command via a child process
+     *
+     * @param util {string}
+     * @param utilArgs {ReadonlyArray<string>}
+     */
+    public execCmd(util: string, utilArgs: ReadonlyArray<string>): Promise<ChildProcess> {
+        logger.info(`window handler: execCmd: util: ${util} utilArgs: ${utilArgs}`);
+        return new Promise<ChildProcess>((resolve, reject) => {
+            return execFile(util, utilArgs, (error: ExecException | null) => {
+                logger.info(`window handler: execCmd: error: ${error}`);
```

Comment:
Avoid printing the error when operation successful:
```suggestion
                if (error) {
                    logger.info(`window handler: execCmd: error: ${error}`);
                }
```